src/styles/index.css

```diff
@@ -0,0 +1,130 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card_info {
+  text-align: left;
+  padding: 0 16px;
+}
+
+.card_imac--img {
+  margin-top: 32px;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+  padding: 0 19px;
+}
+
+.card_text--name {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card_text--code {
+  margin-top: 4px;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card_stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars_number {
+  margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card_price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card_price--name {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card_price--number {
+  margin: 0;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+}
+
+.card_button {
+  display: flex;
+  justify-content: center;
+}
+
+.card_button--buy {
+  all: unset;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  color: #fff;
+  background-color: #00acdc;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px 0;
+  cursor: pointer;
+}
+
+.card_button--buy:hover {
+  color: #00acdc;
+  box-sizing: border-box;
```

Comment:
The 'box-sizing' property should not be changed on hover. It should be consistent with the element's normal state. If you need to adjust the border on hover, consider using 'outline' as an alternative to 'border' to avoid affecting the element's dimensions.